communication/contributor-comms/role-handbooks/Social-Media.md

```diff
@@ -13,28 +13,28 @@ Do note that tweets targeting the greater Kubernetes community should potentiall
 
 ## Minimum Skills and Requirements
 
-- Are a member of the [Kubernetes GitHub Org]
+- Are a member of the Kubernetes GitHub Org
 - Familiar with twitter by either having a personal account or managing another account (brand, oss project, employer, local nonprofit, etc)
-- Available to either attend regular Contributor Comms meetings or collaborate asynchronously via the #sig-contribex-comms channelnin the Kubernetes Slack
+- Available to either attend regular Contributor Comms meetings or collaborate asynchronously via the #sig-contribex-comms channel in the Kubernetes Slack
 
 ### Expected Time Investment
 
 Team leads: 3-5 hours per week.
-Team members: 1-3 hour per week.
+Team members: 1-3 hours per week.
 
 ## General Expectations
 
 - Rules for social posts and more can be found in [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
 - Be welcoming, be yourself
-- Understanding, communicating, and setting/adjusting [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
+- Understanding, communicating and setting/adjusting [social guidelines](https://github.com/kubernetes/community/blob/master/communication/contributor-comms/storytelling-resources/social-guidelines.md)
 - Coordinating work to create tweets, ensuring tweet requests are fulfilled
 - Working with other Contributor Comms leads, such as the Event Coordinator Lead, on social media tasks
 - Approving tweet PRs on [kubernetes-sigs/contributor-tweets](https://github.com/kubernetes-sigs/contributor-tweets)
 - Training new leads
 
-The leader(s) of the social media team will be expected to monitor the @K8sContributors handle. Retweets, replies, and responding to replies appropriately fall under the purview of the social media lead. Time-sensitive tweets should also be handled directly by the team lead. Team leads should coordinate with team members to plan tweets in advance, particularly for large contributor-focused events such as KubeCon or Contributor Summit.
```

Comment:
This is not a typo. For more information: https://en.wikipedia.org/wiki/Serial_comma

Reply:
Thanks for the info, I have reverted this change but not sure why it is not showing `outdated`.